oeps/processes/oep-0021-proc-deprecation.rst

```diff
@@ -177,6 +177,13 @@ When proposing a removal, consider the following analysis:
 Timeline
 --------
 
+To communicate a proposed removal, you will need to determine the *earliest*
+Open edX named release that could no longer have the removed functionality.
+Choosing the earliest named release where removal could be complete is the
```

Comment:
Do we already have all predefined names for the upcoming releases? Or we could add a letter reference instead of a name, like `T - named` release?

Reply:
Thanks @idegtiarov. I had done exactly what you mentioned in the help text in the template: https://github.com/openedx/.github/blob/a4b31ac53192e60b82ac79d70e708738187c43a6/.github/ISSUE_TEMPLATE/depr-ticket.yml. I can add a note here as well. I can also clarify that we'll use the CUT dates.

Reply:
@idegtiarov @kdmccormic: Would you be up for reviewing the latest commit related to this comment thread? https://github.com/openedx/open-edx-proposals/pull/404/commits/a4e84a73aa71156656b17227fbacda469227be93. Note that the template also has a help message that includes more specific examples, as noted in my last comment.

Reply:
Looks good to me @robrap 👍🏻

Reply:
Thanks.